addons/pvr.mediaportal.tvserver/src/Socket.cpp

```diff
@@ -138,7 +138,7 @@ bool Socket::bind ( const unsigned short port )
   _sockaddr.sin_addr.s_addr = INADDR_ANY;  //listen to all
   _sockaddr.sin_port = htons( port );
 
-  int bind_return = ::bind(_sd, (sockaddr*)(&_sockaddr), sizeof(_sockaddr));
+  int bind_return = MPTV::bind(_sd, reinterpret_cast<sockaddr*>(&_sockaddr), sizeof(_sockaddr));
```

Comment:
I agree that it looks wrong. At least on osx when enabling c++11 Standard it seems that alot of Std:: stuff is Moved into the global namespace - and sadly Std:: has a bind Method aswell which for some reason Takes priority. I was not Able to fix the call to bind in a way that the Compiler would pick the correct Socket related bind. I even think that this should also be a Problem when compiling with c++11 and clang on Linux tbh. I am Open for suggestions of Course. Calling bind from the mptv namespace in the end makes the Compiler use the Socket bind (because there is no bind with matching signature in that namespace i guess)